Formula/fx.rb

```diff
@@ -18,14 +16,13 @@ class Fx < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "91533eff62e86951a9b67f7d9ced7673627add44ac66320199f8336b6fc576ad"
   end
 
-  depends_on "node"
+  depends_on "go" => :build
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    system "go", "build", *std_go_args
   end
 
   test do
-    assert_match "bar", shell_output("echo '{\"foo\": \"bar\"}' #{bin}/fx .foo")
+    assert_equal "42", shell_output("echo 42 | #{bin}/fx").strip
```

Comment:
```suggestion
    assert_equal "42", pipe_output(bin/"fx", 42).strip
```
Your old test didn't work because `fx` doesn't handle `.json` files that are missing a new line before the EOF.

Not sure if the specification requires the new line, but it seems to handle it fine on macOS.

Reply:
Will take a look.